### PR TITLE
[lua] untargetable -> setUntargetable

### PR DESCRIPTION
**Pull request by zach2good**
**_I affirm:_**
- [x] I have paid attention to this example and will edit again if need be to not break the formatting, or I will be ignored
- [x] I have read and understood the [Contributing Guide](https://github.com/LandSandBoat/server/blob/base/CONTRIBUTING.md)
- [x] I've _**tested my code and the things my code has changed**_ since the last commit in the PR, and will test after any later commits

## What does this pull request do?

Renames methods and bindings from untargetable -> setUntargetable

## Steps to test these changes

CTRL+SHIFT+F to find the old calls and bindings, make sure there aren't any warnings on startup

### CODE REVIEW

File: src/map/lua/lua_baseentity.cpp [hunk 1]
```diff
@@ -14505,7 +14505,7 @@ void CLuaBaseEntity::Register()
     SOL_REGISTER("setAggressive", CLuaBaseEntity::setAggressive);
     SOL_REGISTER("setTrueDetection", CLuaBaseEntity::setTrueDetection);
     SOL_REGISTER("setUnkillable", CLuaBaseEntity::setUnkillable);
-    SOL_REGISTER("untargetable", CLuaBaseEntity::untargetable);
+    SOL_REGISTER("setUntargetable", CLuaBaseEntity::setUntargetable);
```
Comment (claywar) [inline src/map/lua/lua_baseentity.cpp:14531]: Though unused, should we go ahead and create a getUntargetable lua binding to match?

**Review by claywar — APPROVED** [review]

File: src/map/lua/lua_baseentity.cpp (hunk 1 above)
Comment (zach2good) [reply src/map/lua/lua_baseentity.cpp:14531]: I did think about that, will do

Comment (Gradius2) [reply src/map/lua/lua_baseentity.cpp:14531]: Yes, I'm fully updated and of course, compiled thereafter.

**Comment by zach2good** [comment]
https://github.com/LandSandBoat/server/runs/6717730179?check_suite_focus=true

None of those errors in our comprehensive Windows and Linux startup checks.

**Comment by Gradius2** [comment]
I updated and compiled multiple times over the last month.

This is the first time I saw those errors, it has nothing to do with the link you provided.

I compiled just now, same errors.  I wonder what could be have gone wrong.

**Comment by Gradius2** [comment]
I fully updated the /scripts folder by hand.  Same issue.  Could be due VS2022 ?  I'm running the last updated VS2022.

However is weird, never got such error before.

**Comment by TeoTwawki** [comment]
"by hand"? proper updating is just using git pull

And what the link has to do with it is showing you a normal run doesn't have those errors. The 1st error message is telling you that function doesn't exist. you've done something wrong and it doesn't exist in your source but you have scripts calling it.

**Comment by Xaver-DaRed** [comment]
This isnt a place to have a conversation
And this only goes to prove that, up to now, you were not building properly. I'm getting the same kernel error, witch means there is no way you were building up to this point

**Pull request merged**